arch/arm/insert_string_acle.c

```diff
@@ -12,15 +12,21 @@
 #include "../../zbuild.h"
 #include "../../deflate.h"
 
+#if defined(__aarch64__)
```

Comment:
I am thinking instead of duplicating this block, perhaps it could be placed in `acle_intrins.h` or `arm_features.h`.
Neither is a perfect match, but I think either would be OK.

Reply:
I think `acle_intrins.h` would be better so it doesn't pollute other unrelated source files unnecessarily...